sample-dcrwallet.conf

```diff
@@ -10,6 +10,12 @@
 ; Use simnet (cannot be used with testnet=1).
 ; simnet=0
 
+; Set the private wallet passphrase. This option may reduce security,it should
```

Comment:
Sorry, a few more suggestions (want to get this right!)

Since the real reason to set this config option is to unlock the wallet at startup and prevent the private passphrase prompt from ever showing, we should mention that first.

There needs to be a space after the comma.

We should also note that this overrides setting --promptpass and can be used to run the wallet with the ticketbuyer without prompting at startup.